app/controllers/comments_controller.rb

```diff
@@ -1,13 +1,42 @@
 class CommentsController < ApplicationController
   before_action :authenticate_user!
+
 def create
   @post = Post.find(params[:post_id])
   @post.comments.create(comment_params.merge(:user => current_user))
   redirect_to post_path(@post)
 end
 
+def edit
+  @comment = Comment.find(params[:comment_id])
+end
+
+def update
+  @comment = Comment.find(params[:comment_id])
+  @comment.update_attributes(comment_params)
+  redirect_to post_path(@post)
+end
+
+def destroy
+  @post = comment.post
+  comment.destroy 
+  redirect_to post_path(@post)
+end
+
 private
 
+def require_comment
+  render_not_found unless @comment
+end
+
+def comment
```

Comment:
I saw that you used: 
def require_current_comment
    render_not_found unless current_comment
  end
in your Yelp clone, can you help me understand why I'd want to rename this method to current_comment?